definitions/npm/redux-reactors_v1.x.x/flow_v0.95.x-/redux-reactors_v1.x.x.js

```diff
@@ -0,0 +1,55 @@
+/* Helper types from 'redux'
```

Comment:
Happy to hear thoughts on alternatives to this.

For context, this package relies upon the types exported from `redux`.  Ideally we don't copy+paste these (brittle) and pull from those other types directly.